src/activities/GetPortal.ts

```diff
@@ -0,0 +1,25 @@
+import Portal from "@arcgis/core/portal/Portal";
+import type { IActivityHandler } from "@geocortex/workflow/runtime";
+
+interface GetPortalOutputs {
+    /**
+     * @description The default portal instance.
+     */
+    portal: __esri.Portal;
+}
+
+/**
+ * @category ArcGIS Maps SDK for JavaScript
+ * @clientOnly
+ * @defaultName portal
+ * @description Gets the default portal instance.
```

Comment:
Gets the default Portal instance.  Portal should be capitalized for descriptions

Reply:
Done